### PR TITLE
Webdocs

### PR DESCRIPTION
**Pull request by hpcowley**
Added documentation for large_graph

### CODE REVIEW

**Comment by j6k4m8** [comment]
Looks like this includes the #40 changes as well, no?

**Comment by hpcowley** [comment]
Ack, yes. Sorry @j6k4m8 . Not sure how that happened.

**Comment by j6k4m8** [comment]
Add `large_graph` to the changelog for `0.1.3` so there's a record of this, and then merge :)

File: docs/_layouts/docpage.html [hunk 1]
```diff
@@ -18,6 +18,7 @@
 			<li class="list-group-item"><a href="{{ site.baseurl }}/docs/axes"><code>axes</code></a></li>
 			<li class="list-group-item"><a href="{{ site.baseurl }}/docs/fibers"><code>fibers</code></a></li>
 			<li class="list-group-item"><a href="{{ site.baseurl }}/docs/graph"><code>graph</code></a></li>
+			<li class="list-group-item"><a href="{{ site.baseurl }}/docs/largegraph"><code>large graph</code></a></li>
```
Comment (j6k4m8) [inline docs/_layouts/docpage.html]: this text should match the API (i.e. `large_graph` w underscore)

File: docs/docs/largegraph.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+```python
+pytri.large_graph(data, [radii], [colors])
+```
+
+### Arguments
+* `data`: A [Networkx](https://networkx.github.io/documentation/networkx-1.10/overview.html) graph, passed in as [JSON](https://networkx.github.io/documentation/networkx-1.10/reference/readwrite.json_graph.html) or as a Networkx graph object.
+    * Graph must have nodes with positions as attributes. These positions may be specified as a tuple or a dictionary:
+        * `graph.add_node(id, pos=(x_pos, y_pos, z_pos))`
+        * `graph.add_node(id, pos ={'x': x_pos, 'y': y_pos, 'z': z_pos})`
+* `radii` (Optional): Radius of nodes in the visualization. Visualiation will render as cubes, with default size of 2pts.
```
Comment (j6k4m8) [inline docs/docs/largegraph.md]: This can be an array or a scalar, correct?

**Comment by j6k4m8** [comment]
Closing, see #71

**Pull request closed without merging**